operations/_scripts/generate/generate_identifier_supershort.sh

```diff
@@ -3,7 +3,7 @@
 set -e
 
 if [ -z "$AWS_RESOURCE_IDENTIFIER" ]; then
-  GITHUB_IDENTIFIER="$(echo $($GITHUB_ACTION_PATH/operations/_scripts/generate/generate_identifier.sh) | tr '/' '-' )"
+  GITHUB_IDENTIFIER="$($GITHUB_ACTION_PATH/operations/_scripts/generate/generate_identifier.sh)"
```

Comment:
We are calling `generate_identifier.sh` multiple times, anyway to set the variable once and load it into the env for other scripts?